rrrspec-client/lib/rrrspec/client/slave_runner.rb

```diff
@@ -69,15 +74,39 @@ def work
           soft_timeout_sec, hard_timeout_sec = spec_timeout_sec(task)
 
           formatter = RedisReportingFormatter
+
+          AfterConfiguration do |config|
```

Comment:
Same here. It's odd that there's an abstraction on the runner so that this slave code is decoupled from RSpec, but the reporting part is tightly coupled to Cucumber.